### ISSUE TITLE
selinux_permissive no_reload option should be renamed/deprecated

### ISSUE THREAD

**Issue by orev** [ansible/ansible-modules-extras#2321]
##### ISSUE TYPE
- Feature Idea
##### COMPONENT NAME

selinux_permissive module
##### ANSIBLE VERSION

```
ansible 2.0.2.0
```
##### OS / ENVIRONMENT

CentOS 7
##### SUMMARY

The "no_reload" option is badly named.  It starts with a "negative" concept (no_), then you set either true or false to enable/disable the negative concept.  This double-negative is confusing and should be renamed to use a more obvious/positive concept.

Name should be changed to "reload" with default "true".  The no_reload option could still be supported as deprecated to prevent breakage in current playbooks.

**Comment by mscherer** [comment]
Seems sensible. I use no_reload, because that the interface provided by selinux binding, but this should be better.

**Comment by ansibot** [comment]
@mscherer, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by mscherer** [comment]
So another issue I have is that 'reload' is a built-in function. I am not 100% sure on how to handle that.

**Comment by ansibot** [comment]
@mscherer, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@mscherer, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by mscherer** [comment]
@ansibot I made a PR about it

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pullrequests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pullrequest to the ansible/ansible repo.

**Comment by ansibot** [comment]
This issue was migrated to https://github.com/ansible/ansible/issues/29318
